extensions/agroal/deployment/src/main/java/io/quarkus/agroal/deployment/AgroalProcessor.java

```diff
@@ -344,14 +345,13 @@ private String resolveDriver(String dataSourceName, String dbKind,
             }
         }
 
-        throw new ConfigurationException("Unable to find a JDBC driver corresponding to the database kind '"
-                + dbKind + "' for the "
-                + (DataSourceUtil.isDefault(dataSourceName) ? "default datasource"
-                        : "datasource '" + dataSourceName + "'")
-                + ". Either provide a suitable JDBC driver extension, define the driver manually, or disable the JDBC datasource by adding "
-                + (DataSourceUtil.isDefault(dataSourceName) ? "'quarkus.datasource.jdbc=false'"
-                        : "'quarkus.datasource." + dataSourceName + ".jdbc=false'")
-                + " to your configuration if you don't need it.");
+        throw new ConfigurationException(String.format(
+                "Unable to find a JDBC driver corresponding to the database kind '%s' for the %s (available: '%s'). "
```

Comment:
I'm not entirely sure what the "available: [list of dbkinds]" stands for ?

Reply:
The possible values, in case there is a typo in the configuration

Reply:
In https://github.com/quarkusio/quarkus/issues/30701#issuecomment-1410757424 the dbKind was configured as `postgres` but the correct value is `postgresql`

Reply:
yes I understood you, but it could be clearer I think.

Reply:
Sure, since this PR was already merged, feel free to send another PR ;)